modules/tinymce/src/core/test/ts/browser/content/InsertContentTest.ts

```diff
@@ -859,4 +859,65 @@ describe('browser.tinymce.core.content.InsertContentTest', () => {
       TinyAssertions.assertContent(editor, '<p>a<svg><circle cx="50" cy="50" r="40" stroke="black" stroke-width="3" fill="red"><desc><p>hello</p></desc></circle></svg>b</p>');
     });
   });
+
+  context('${caret) placeholder of caret position after insert', () => {
+    const hook = TinyHooks.bddSetupLight<Editor>({
```

Comment:
I don't think you need this one also if you have one hook in the parent scope then one in the sub scope I think there will be two editors at the same time I realize now that I did the same mistake on the svg element I could have put the extended_valid_elements on the parent editor and use that.
So I suggest re-using the hook from the parent scope it has indent: false but some custom elements.

Reply:
If I remove this TinyHooks.bddSetupLight call the tests start failing since hook.editor().getBody() returned null.
![image](https://github.com/tinymce/tinymce/assets/44773772/52fc1688-10db-43d3-ba3e-ffaeef1d1613)

But if remove that one for the upper SVG tests as well, the tests are passed. But I don't like to move  extended_valid_elements for svg to the parent scope since it looks like something specific for svg test context which creeps out and clutters the parent scope, reducing readability.

Reply:
Right maybe we should split these out then into separate files. The file is beginning to become quite large. Like a new InsertContentCaretPlaceholderTest.ts then we might move out the SVG test to InsertContentSvgTest.ts or something.

Reply:
We could just add the new placeholder test file now then have a code cleanup task to split it apart more.

Reply:
I've created a separate `insert` folder for InsertContent related tests.